### ISSUE TITLE
Addition custom actions and locale params for request_sms and request_voice

### ISSUE THREAD

**Issue by RobWelbourn** [evilpacket/node-authy#29]
@josh-authy In addition to the `force` parameter, it would be useful to include the optional parameters `action`, `action_message` and `locale` to the request_sms and request_voice methods.

**Comment by manumouton** [comment]
I totally agree, we've been hacking the code in order to force the call to be performed in English. 
Do you think you could republish a package with this slight modification?

`Authy.prototype.request_call = function (id, force, callback) {
    var qs = {};

    if (arguments.length > 2) {
        qs.force = force;
    } else {
        callback = force;
    }
    qs.locale = 'en' ;

    this._request("get", "/protected/json/call/" + querystring.escape(id), {}, callback, qs);
};`
